homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,30 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+
+    int a = 0;
+    int b = 1;
+
+    for( int fibonacci_sequence_counter = 0; fibonacci_sequence_counter < sequence; fibonacci_sequence_counter++ )
+    {
+        b += a;
+        a = b - a;
+    }
+
+    return a;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+
+    if ( sequence == 0)
+    {
+        return 0;
+    }
+
+    if ( sequence < 3)
+    {
+        return 1;
+    }
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

You avoid 2 if